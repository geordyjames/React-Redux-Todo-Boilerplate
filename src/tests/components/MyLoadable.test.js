import React from 'react';
import { shallow } from 'enzyme';
import MyLoadable from '../../components/MyLoadable';

test('should render MyLoadable with empty props correctly', () => {
  const wrapper = shallow(<MyLoadable />);
  expect(wrapper).toMatchSnapshot();
});

test('should render MyLoadable with opts props correctly', () => {
  const wrapper = shallow(<MyLoadable opts={{loader: () => {}}} />);
  expect(wrapper).toMatchSnapshot();
});