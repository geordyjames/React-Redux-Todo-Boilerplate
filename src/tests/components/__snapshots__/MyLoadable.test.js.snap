// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyLoadable with empty props correctly 1`] = `<Component />`;

exports[`should render MyLoadable with opts props correctly 1`] = `<Component />`;
